openedx/adg/lms/applications/models.py

```diff
@@ -243,7 +243,7 @@ def is_education_experience_completed(self):
         """
         Check user completed the education and experience step in user application
         """
-        return self.is_education_completed and self.is_work_experience_completed and self.background_question
+        return self.is_education_completed and self.is_work_experience_completed and bool(self.background_question)
```

Comment:
Didn't we discuss to include a check for references here as well?

Reply:
That will be added while resolving conflicts
As I do not have access to that yet